integration/setup/setup.go

```diff
@@ -302,3 +212,85 @@ func installCredential(h *framework.Host) error {
 
 	return nil
 }
+
+func installHostPeerVPC(config Config) (string, error) {
+	log.Printf("creating Host Peer VPC stack")
+
+	os.Setenv("AWS_ROUTE_TABLE_0", env.AWSRouteTable0())
+	os.Setenv("AWS_ROUTE_TABLE_1", env.AWSRouteTable1())
+	os.Setenv("CLUSTER_NAME", env.ClusterID())
+	stackName := "host-peer-" + env.ClusterID()
+	stackInput := &cloudformation.CreateStackInput{
+		StackName:        aws.String(stackName),
+		TemplateBody:     aws.String(os.ExpandEnv(e2etemplates.AWSHostVPCStack)),
+		TimeoutInMinutes: aws.Int64(2),
+	}
+	_, err := config.AWSClient.CloudFormation.CreateStack(stackInput)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+	err = config.AWSClient.CloudFormation.WaitUntilStackCreateComplete(&cloudformation.DescribeStacksInput{
+		StackName: aws.String(stackName),
+	})
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+	describeOutput, err := config.AWSClient.CloudFormation.DescribeStacks(&cloudformation.DescribeStacksInput{
+		StackName: aws.String(stackName),
+	})
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+	var vpcPeerID string
+	for _, o := range describeOutput.Stacks[0].Outputs {
+		if *o.OutputKey == "VPCID" {
+			vpcPeerID = *o.OutputValue
+			break
+		}
+	}
+	log.Printf("created Host Peer VPC stack")
```

Comment:
We should use our logging and the appropriate format.

Reply:
After merging I'm going to inject logger into the Config so we can get rid of mixed logging style.

Reply:
That is nice. Using the configured logger will be super cool.